content/src/roadmaps/add-ons/public-record-filing.json

```diff
@@ -1,14 +1,9 @@
 {
   "roadmapSteps": [
-    {
-      "step": 2,
-      "weight": 19,
-      "task": "search-business-name"
-    },
     {
       "step": 2,
       "weight": 21,
-      "task": "form-business-entity"
+      "task": "form-business-entity-nj"
```

Comment:
I feel myself getting very confused with the quantity of "form business entity" tasks and what they all do... it feels like it needs a high-level assessment at some point